libs/category/src/filters/public_api.ts

```diff
@@ -8,6 +8,7 @@ export * from './to-request/public_api';
 export * from './type/range/public_api';
 export * from './type/equal/public_api';
 export * from './validators/public_api';
```

Comment:
I don't think the errors should be exported here, they should only be from the root errors API.

Reply:
moved export

Reply:
I misunderstood what was in the API of filter errors, my mistake. They can be exported from the filters public API.

Reply:
reverted